presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/converter/GlueInputConverter.java

```diff
@@ -84,9 +84,6 @@ public static PartitionInput convertPartition(Partition partition)
 
     private static StorageDescriptor convertStorage(Storage storage, List<Column> columns)
     {
-        if (storage.isSkewed()) {
```

Comment:
are we retaining this information in the returned `StorageDescriptor`?

Reply:
no

Reply:
> are we retaining this information in the returned StorageDescriptor?

good point. changed the code.